js/tests/unit/sdk.test.js

```diff
@@ -196,29 +205,50 @@ test('video element should take all available space', () => {
 })
 
 test('rotate video element', () => {
-    const stream = new AnboxStream(sdkOptions)
-    stream._webrtcManager.sendControlMessage = jest.fn(() => { return true })
+    let test_video_rotation = function(options) {
+      const stream = new AnboxStream(options)
+      stream._webrtcManager.sendControlMessage = jest.fn(() => { return true })
+
+      const video = document.createElement('video')
+      video.id = stream._videoID
+      video.__defineGetter__('videoWidth', () => 500);
+      video.__defineGetter__('videoHeight', () => 1000);
+      const container = document.getElementById(sdkOptions.targetElement)
+      container.__defineGetter__('clientWidth', () => 1000);
+      container.__defineGetter__('clientHeight', () => 1000);
+      container.appendChild(video)
+
+      let visualElement = video
+      if (options.experimental.upscaling.enabled) {
+          const canvas = document.createElement('canvas')
+          canvas.id = stream._canvasID
+          container.appendChild(canvas)
+          visualElement = canvas
+
+          stream._streamCanvas = {
+            resize() {},
+            stop() {},
+          }
+      }
 
-    const video = document.createElement('video')
-    video.id = stream._videoID
-    video.__defineGetter__('videoWidth', () => 500);
-    video.__defineGetter__('videoHeight', () => 1000);
-    const container = document.getElementById(sdkOptions.targetElement)
-    container.__defineGetter__('clientWidth', () => 1000);
-    container.__defineGetter__('clientHeight', () => 1000);
-    container.appendChild(video)
-    stream._onResize()
-    expect(stream.getCurrentOrientation()).toEqual('portrait')
+      stream._onResize()
+      expect(stream.getCurrentOrientation()).toEqual('portrait')
 
-    stream.rotate('landscape')
+      stream.rotate('landscape')
 
-    expect(video.style.transform).toEqual('rotate(90deg)')
-    expect(stream._webrtcManager.sendControlMessage.mock.calls.length).toEqual(1)
-    expect(stream._webrtcManager.sendControlMessage.mock.calls[0][0]).toEqual('screen::change_orientation')
-    expect(stream._webrtcManager.sendControlMessage.mock.calls[0][1]).toEqual({orientation: 'landscape'})
-    expect(stream.getCurrentOrientation()).toEqual('landscape')
+      expect(visualElement.style.transform).toEqual('rotate(90deg)')
+      expect(stream._webrtcManager.sendControlMessage.mock.calls.length).toEqual(1)
+      expect(stream._webrtcManager.sendControlMessage.mock.calls[0][0]).toEqual('screen::change_orientation')
+      expect(stream._webrtcManager.sendControlMessage.mock.calls[0][1]).toEqual({orientation: 'landscape'})
+      expect(stream.getCurrentOrientation()).toEqual('landscape')
 
-    expect(() => stream.disconnect()).not.toThrow()
+      expect(() => stream.disconnect()).not.toThrow()
+    };
+
+    test_video_rotation(sdkOptions)
+
+    sdkOptions.experimental.upscaling.enabled=true;
```

Comment:
Looks like we miss a style checker:

```suggestion
    sdkOptions.experimental.upscaling.enabled = true;
```

Reply:
We only run ESLint against the anbox-stream-sdk.js file, not the test files under js/tests/unit:
- callbacks.test.js
- sdk.test.js
- input-events.test.js
- stats.test.js

Yes, it would be nice if we could enable ESLint for those files as well. 
Since it's gonna be the first time running ESLint for the unit test files, the change to those test file might be big. Hence let's not mix up the scope of this PR.
If you give a +1, I can send another PR to do this.  How does that sound?